daml-assistant/daml-helper/test/DA/Daml/Helper/Test/Tls.hs

```diff
@@ -21,45 +22,87 @@ main = do
     damlHelper <- locateRunfiles (mainWorkspace </> "daml-assistant" </> "daml-helper" </> exe "daml-helper")
     sandbox <- locateRunfiles (mainWorkspace </> "ledger" </> "sandbox" </> exe "sandbox-binary")
     certDir <- locateRunfiles (mainWorkspace </> "daml-assistant" </> "daml-helper")
-    withTempFile $ \portFile ->
-        withBinaryFile nullDevice ReadWriteMode $ \devNull ->
-        defaultMain $ withResource (createSandbox devNull sandbox portFile (certDir </> "server.crt", certDir </> "server.pem") (certDir </> "ca.crt")) destroySandbox $ \getSandbox ->
+    defaultMain $
         testGroup "TLS"
-            [ testCase "Party management" $ do
-                  p <- sandboxPort <$> getSandbox
-                  let ledgerOpts =
-                          [ "--host=localhost", "--port", show p
-                          , "--cacrt", certDir </> "ca.crt"
-                          , "--pem", certDir </> "client.pem"
-                          , "--crt", certDir </> "client.crt"
-                          ]
-                  out <- readProcess damlHelper
-                      ("ledger" : "list-parties" : ledgerOpts)
-                      ""
-                  assertInfixOf "no parties are known" out
-                  out <- readProcess damlHelper
-                      ("ledger" : "allocate-party" : "Alice" : ledgerOpts)
-                      ""
-                  assertInfixOf "Allocated 'Alice' for 'Alice'" out
-                  out <- readProcess damlHelper
-                      ("ledger" : "list-parties" : ledgerOpts)
-                      ""
-                  assertInfixOf "PartyDetails {party = 'Alice', displayName = \"Alice\", isLocal = True}" out
-            -- TODO (MK) Once we have a ledger server (e.g. sandbox) that supports
-            -- TLS without client auth, we should test this here as well.
-            ]
+           [ withSandbox sandbox certDir "none" $ \getSandboxPort ->
```

Comment:
I’ve simplified the tests somewhat to only make one request. Given that the purpose of those tests is to test TLS and not the `daml ledger` commands everything else seems redundant.